### PR TITLE
banner: 100M vaccinations

### PR DESCRIPTION
**Pull request by pnavarrc**
[Trello](https://trello.com/c/xF3CfeRe/1214-vaccination-homepage-takeover) - 100M vaccinations banner

**Mobile**
screenshot of the banner on mobile

**Desktop**
screenshot of the banner on desktop

### CODE REVIEW

**Comment by pnavarrc** [comment]
[Vaccination Homepage Takeover](https://trello.com/c/xF3CfeRe/1214-vaccination-homepage-takeover)

File: src/components/Banner/Vaccinations100M.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    <BannerContainer>
+      <InnerContainer>
+        <Body>
+          <strong>
+            Over 100M people in the US have received at least one dose of a
+            COVID vaccine!{' '}
+          </strong>
+          <span role="img" aria-label="party popper">
+            🎉
+          </span>{' '}
```
Comment (pnavarrc) [inline src/components/Banner/Vaccinations100M.tsx:43]: Storybook warned me of this, emojis need `role="img"` and `aria-label`

Comment (mikelehen) [reply src/components/Banner/Vaccinations100M.tsx:43]: Wow, nice. 😁 

I think maybe this broke the spacing after the emoji though.  Do we need to throw in a `&nbsp;` or something?  Not sure...  
image

**Review by mikelehen — APPROVED** [review]
I think maybe the spacing regressed with the aria-label span?  But otherwise this looks great.  Thanks for cleaning up some code along the way.

**Pull request merged**